### PR TITLE
teams/docs: a vote for Oreoxmt as the committer

### PR DESCRIPTION
**Pull request by TomShawn**
As stated in the title.

Hi https://github.com/orgs/pingcap/teams/tidb-maintainers, PTAL.

/hold

### CODE REVIEW

**Comment by ran-huang** [comment]
+1 👏

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

File: votes/0675-Oreoxmt-as-docs-committer.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Deadline
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
+
```
Comment (lilin90) [inline votes/0675-Oreoxmt-as-docs-committer.md:35]: @TomShawn Please help update the voting result here.

Comment (TomShawn) [reply votes/0675-Oreoxmt-as-docs-committer.md:35]: Done

**Pull request merged**